spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -162,10 +162,10 @@
           allow(OneLogin::RubySaml::Logoutresponse).to receive(:new).and_return(invalid_logout_response)
         end
 
-        it 'redirects to error' do
+        it 'redirects as success even when logout fails, but it logs the failure' do
```

Comment:
> even when logout fails

is reduntant to [`context ' logout_response is invalid'`](https://github.com/department-of-veterans-affairs/vets-api/pull/1750/files#diff-13a3322b35e5f7b1319d70af5d8e470aR160), so this can probably just read "redirects as success and logs the error"